index.html

```diff
@@ -92,7 +130,40 @@ <h3>We put recipes to the <br>test, so you can<br>savor the success!</h3>
     <div class="img2"></div>
   </div>
 </body>
+<!-- Reviews Section -->
+<div class="reviews" id="reviews">
+  <h1>Reviews</h1>
+  <div id="review-list"></div>
+</div>
+
+<script>
+  const reviews = [
+    { username: "Username1", date: "2024-12-12 13:30", stars: 5, review: "Amazing recipes! I tried the Burnt Rice Bowl and it was
+    { username: "Username2", date: "2024-11-24 14:45", stars: 5, review: "Loved it! Perfect and easy for beginners. " },
+    { username: "Username3", date: "2023-09-22 20:15", stars: 4, review: "Great unique recipes but a bit hard to follow" }
+  ];
+
+  function Reviews() {
+    const reviewList = document.getElementById('review-list');
+    reviewList.innerHTML = '';
+    reviews.forEach(review => {
+      const reviewDiv = document.createElement('div');
+      reviewDiv.classList.add('review-items');
+      reviewDiv.innerHTML = `
+        <img src="images/78-786293_1240-x-1240-0-avatar-profile-icon-png.png" alt="Profile Pic" class="profile-pic">
+        <div class="review-content">
+          <h3>${review.username}</h3>
+          <p>Posted on: ${review.date}</p>
+          <p>${'⭐'.repeat(review.stars)}</p>
+          <p>${review.review}</p>
+        </div>
+      `;
+      reviewList.appendChild(reviewDiv);
+    });
+  }

```

Comment:
This needs to be in the individual recipe page, and not the landing page. The JavaScript code should be in its JS file as well.

Reply:
This too, 
![image](https://github.com/user-attachments/assets/1d0ba60d-51e4-4aa8-ab5f-337752316daa)

This is not supposed to get displayed in the landing page